docker-compose.yml

```diff
@@ -174,9 +174,18 @@ services:
     image: appflowyinc/appflowy_ai:${APPFLOWY_AI_VERSION:-latest}
     environment:
       - OPENAI_API_KEY=${AI_OPENAI_API_KEY}
-      - APPFLOWY_AI_SERVER_PORT=${AI_SERVER_PORT}
-      - APPFLOWY_AI_DATABASE_URL=${AI_DATABASE_URL}
-      - APPFLOWY_AI_REDIS_URL=${AI_REDIS_URL}
+      - ANTHROPIC_API_KEY=${AI_ANTHROPIC_API_KEY}
```

Comment:
This would actually cause the current AI container to fail, because the service itself still expects the old variable name, instead of the updated one.